app/assets/stylesheets/config/_colors.css

```diff
@@ -276,6 +276,24 @@
   --indicator-info-bg: var(--accent-brand);
   --indicator-info-color: rgb(var(--white));
 
+  /***********************************************
+  ** Colors **************************************
+  ***********************************************/
+
+  --color-primary: rgb(var(--grey-900));
+  --color-secondary: rgb(var(--grey-500));
```

Comment:
Is there a more descriptive name we can give for these? Or maybe just a description of how they're intended to be used... atm I'm not sure what the use cases for these would be

Reply:
We often use 2 shades of "base" color for tons of things: `base-100` (or `base`) for primary text and `base-60` for anything that we want to de-emphasize a little. I actually pretty consciously decided to use these very generic name because their purpose is to replace ale the places where we were too specific/granular with naming. 

So these literally are simply two main colors I'd like to be used for content. I was considering using something a bit more descriptive, like `text-primary` or `typography-primary` but we'd also use these for other elements like icons. Hence this very generic naming.

But I'm very option for suggestions.

Reply:
That makes sense - maybe a wee inline comment in the file would be cool, with what you mentioned here about 'base color' vs 'slightly de-emphasized color'?